Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/Program.cs

```diff
@@ -6,7 +6,19 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var journal = new Journal();
+            var sendEmail = new SendEmail();
+            sendEmail.ValidateEmailSettings();
+            Console.WriteLine("Please enter your name:");
+            var userName = GetWelcomeMessage();
```

Comment:
Should you be getting a username from a `GetWelcomeMessage` method?
Also, why do you need to get the welcome message. Maybe you can just print the welcome message instead?

Reply:
69e6885dcf4842a3f0947fe5ce6d8d38b698c211 made fixes for all issues on that commit